microsoft/iot-cardboard-js
# Error states / Scene list responsive / Fix useCallback console error

### Summary of changes 🔍 
Added Error states for:
- No Scenes
- No Container selected
- No ADT instance selected

Plus:
- Modified scene list to be responsive
- Removed an error caused by a useCallback inside a conditional.

### Testing 🧪
Scene list:
- Open Scene page with items
- Shrink browser window
- See scrollbar show up and Scene list not hide behind overflow

Error states:
<Stories for testing error states pending>

### Checklist ✔️
